dashboard/templates/files.html

```diff
@@ -1,23 +1,19 @@
+{% extends 'base.html' %}
```

Comment:
we can delete this file as not needed

Reply:
Okay great, I'll delete it in a separate cleanup. @zqian are there other files that can be deleted?

Reply:
Pushyami is right that this file is OK to be removed. But please do not delete others!